src/misc.cpp

```diff
@@ -361,7 +361,7 @@ void std_aligned_free(void* ptr) {
 /// The returned pointer is the aligned one, while the mem argument is the one that needs
 /// to be passed to free. With c++17 some of this functionality could be simplified.
 
-#if defined(_WIN64)
```

Comment:
On my box, I can allocate up to 1750 MB with large pages using the Win32 build. I think 2048 MB is the maximum working set on Windows when the 3GB mode is not enabled in the exe. (I don't think it is enabled in stockfish.)

Outputs:

**32-bit**

```
C:\Users\skiminki\Desktop>stockfish
Stockfish 200920 by the Stockfish developers (see AUTHORS file)
info string Hash table allocation: Windows large pages used.
compiler

Compiled by g++ (GNUC) 7.3.0 on MinGW32
Compilation settings include:  32bit SSE41 SSSE3 SSE2 POPCNT MMX
__VERSION__ macro expands to: 7.3-posix 20180312

```
**64-bit**

```
C:\Users\skiminki\Desktop>stockfish
Stockfish 200920 by the Stockfish developers (see AUTHORS file)
info string Hash table allocation: Windows large pages used.
compiler

Compiled by g++ (GNUC) 7.3.0 on MinGW64
Compilation settings include:  64bit BMI2 AVX2 SSE41 SSSE3 SSE2 POPCNT
__VERSION__ macro expands to: 7.3-posix 20180312
```